record_tx_strategy_outgoing_tx.go

```diff
@@ -17,7 +17,7 @@ type outgoingTx struct {
 
 func (strategy *outgoingTx) Execute(ctx context.Context, c ClientInterface, opts []ModelOps) (*Transaction, error) {
 	logger := c.Logger()
-	logger.Info(ctx, fmt.Sprintf("OutgoingTx.Execute(): start, TxID: %s", strategy.TxID()))
+	logger.Info().Msgf("OutgoingTx.Execute(): start, TxID: %s", strategy.TxID())
```

Comment:
`OutgoingTx.Execute()` Can be added to `Str()`

Reply:
Removed function name from logs